test/e2e/operator_groups_e2e_test.go

```diff
@@ -105,39 +103,33 @@ func TestOperatorGroup(t *testing.T) {
 
 	oldCommand := patchOlmDeployment(t, c, otherNamespaceName)
 
-	log.Debug("Creating CSV")
+	t.Log("Creating CSV")
 	aCSV := newCSV(csvName, testNamespace, "", *semver.New("0.0.0"), nil, nil, newNginxInstallStrategy("operator-deployment", nil, nil))
 	createdCSV, err := crc.OperatorsV1alpha1().ClusterServiceVersions(testNamespace).Create(&aCSV)
 	require.NoError(t, err)
 
-	log.Debug("Creating operator group")
+	t.Log("Creating operator group")
 	operatorGroup := v1alpha2.OperatorGroup{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "e2e-operator-group",
 			Namespace: testNamespace,
 		},
+		Spec: v1alpha2.OperatorGroupSpec{
+			Selector: metav1.LabelSelector{
+				MatchLabels: matchingLabel,
+			},
+		},
 	}
 	_, err = crc.OperatorsV1alpha2().OperatorGroups(testNamespace).Create(&operatorGroup)
 	require.NoError(t, err)
 	expectedOperatorGroupStatus := v1alpha2.OperatorGroupStatus{
 		Namespaces: []*corev1.Namespace{createdOtherNamespace},
 	}
 
-	// instead of setting the label selector initially, do it here to force immediate reconcile
```

Comment:
Did you take this out because a user would never do this?

Reply:
I took this out because it we shouldn't need to do it. The creation of an OperatorGroup should trigger an immediate reconcile. After taking it out I noticed it wasn't, which highlighted a bug with how we were registering multiple queue informers. This PR also addresses that bug! 🙂

Reply:
Wow, ok. Good job!